rest-api-spec/src/main/resources/rest-api-spec/test/update/90_error.yml

```diff
@@ -0,0 +1,10 @@
+---
+'Misspelled fields get "did you mean"':
+  - do:
+      catch: /\[1:2\] \[UpdateRequest\] unknown field \[dac\] did you mean \[doc\]\?/
```

Comment:
I wanted *some* end to end test and a surprising number of things don't use `ObjectParser` in the server.

Reply:
This PR makes it even more compelling that we should migrate as much as possible away from hand-rolled parsing code - it might be worth a divide-and-rule effort like we did with the HLRC or Streamable->Writeable?

Reply:
Yeah, I think so!